### DISCUSSION TITLE
You can't delete the selected category

### DISCUSSION THREAD

**Discussion by LukaHarambasic** [community/community#3076] [Discussions]
If you are in the editing mode and try to delete a category you get a poup with a dropdown with all categories except the one you wanted to delete.  In the example below I clicked on the trash can for `Q&A`.

![image](https://user-images.githubusercontent.com/22857823/103275161-d2432c00-49c3-11eb-9293-615e790720ea.png)

Thank's for this new feature! :)

Cheers,
Luka

**Comment by panzarino** [comment]
Hey Luka, the purpose of this dropdown is not to select what category to delete. If you still have discussions that exist in the category you're trying to delete, you can use this dropdown to reassign them to a new category.

For example, if you click delete on "Q&A" but there are still some discussions there, you can select "Ideen" from the dropdown in order to have those discussions moved into the "Ideen" category when "Q&A" gets deleted.
